modules/postgresql/main.tf

```diff
@@ -0,0 +1,61 @@
+resource "google_compute_global_address" "main" {
+  provider = google-beta
+
+  name = "${var.prefix}database"
+
+  address       = "10.200.1.0"
+  address_type  = "INTERNAL"
+  network       = var.vpc_network_self_link
+  prefix_length = 24
+  purpose       = "VPC_PEERING"
+}
+
+resource "google_service_networking_connection" "main" {
+  provider = google-beta
+
+  network                 = var.vpc_network_self_link
+  reserved_peering_ranges = [google_compute_global_address.main.name]
+  service                 = "servicenetworking.googleapis.com"
+}
+
+resource "google_sql_database_instance" "main" {
+  provider = google-beta
+
+  name             = "${var.prefix}database"
+  database_version = "POSTGRES_9_6"
```

Comment:
long term we may want to allow a user to set the version for this module but it should be fine hard coded for now

Reply:
Agreed. Let's wait until some one asks for it since there will be an amount of testing required to verify supported versions.